Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/EnhancedSurface.azsli

```diff
@@ -24,11 +39,19 @@ class Surface
     float3 position;            //!< Position in world-space
     float3 normal;              //!< Normal in world-space
     float3 vertexNormal;        //!< Vertex normal in world-space
-    float3 albedo;              //!< Albedo color of the non-metallic material, will be multiplied against the diffuse lighting value
-    float3 specularF0;          //!< Fresnel f0 spectral value of the surface
+    float3 baseColor;           //!< Surface base color
+    float3 metallic;            //!< Surface metallic property
```

Comment:
Why is metallic float3 instead of float?

Reply:
should be float will fix, nice catch